deploy/chart/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: ccm-linode
+description: CCM Helm Chart
```

Comment:
I'd recommend we be a bit more descriptive here and say what this chart does. There are some good examples in https://github.com/prometheus-community/helm-charts/blob/main/charts. We might be able to use something from the README.md like:
"The Linode Cloud Controller Manager (CCM) creates a fully supported Kubernetes experience on Linode."